pkg/meta/base.go

```diff
@@ -819,8 +819,70 @@ func (m *baseMeta) cleanupSlices() {
 	}
 }
 
-func (m *baseMeta) StatFS(ctx Context, totalspace, availspace, iused, iavail *uint64) syscall.Errno {
+func (m *baseMeta) StatFS(ctx Context, totalspace, availspace, iused, iavail *uint64, subdir bool) syscall.Errno {
 	defer m.timeit(time.Now())
+	if st := m.statRootFs(ctx, totalspace, availspace, iused, iavail); st != 0 {
+		return st
+	}
+	if m.root == RootInode || !subdir {
+		return 0
+	}
+	var usage *Quota
+	var leftSpace, leftInodes int64 = -1, -1
```

Comment:
use availspace as initial value

Reply:
We need decide whether override `totalspace` or `iused` based on `leftSpace` or `leftInodes` is less than 0

Reply:
My fault, fixed